bazel/grpc_deps.bzl

```diff
@@ -8,6 +8,11 @@ def grpc_deps():
         actual = "@com_github_nanopb_nanopb//:nanopb",
     )
 
+    native.bind(
```

Comment:
The changes in this file seem like they're intended to support bazel, but don't we need corresponding changes in build.yaml to support the other build systems?

Reply:
Okay, as long as we have a viable story that will eventually allow this to work with both, that's fine.